semeio/workflows/misfit_preprocessor/config.py

```diff
@@ -221,17 +221,28 @@ def _inject_default_t(config):
 AUTO_SCALE_SCHEMA = {
     cs.MetaKeys.Type: cs.types.NamedDict,
     cs.MetaKeys.Description: (
-        "The {sc} clustering is supporting multiple parameters."
-    ).format(sc=AUTO_SCALE),
+        f"The {AUTO_SCALE} clustering is supporting multiple parameters. The "
+        "the job will first run pca analysis on all the data to determine the number "
+        "of components. Then spearman correlation is performed on the simulated data"
+        " to create a correlation matrix. Clustering is then performed on the "
+        "correlation matrix using the input from the first pca analysis, where the "
+        "number of clusters equals the number of components. Then pca is performed "
+        "on the individual clusters. Observations in each cluster is then scaled with "
+        "the same value."
+    ),
     cs.MetaKeys.Content: {_LINKAGE: _LINKAGE_SCHEMA},
 }
 
 
 SPEARMAN_CORRELATION_SCHEMA = {
     cs.MetaKeys.Type: cs.types.NamedDict,
     cs.MetaKeys.Description: (
-        "The {sc} clustering is supporting multiple parameters."
-    ).format(sc=SPEARMAN_CORRELATION),
+        f"The {SPEARMAN_CORRELATION} clustering supports multiple parameters. The "
+        "the job will first run spearman correlation on the simulated data to create "
+        "a correlation matrix. Clustering is then performed on the correlation matrix "
+        "before the pca is performed on the individual clusters. Observations in each "
+        "cluster is then scaled with the same value."
```

Comment:
```suggestion
        "cluster are then scaled with the same value."
```
Subject is observations?